src/Components/Header/__test__/Header.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+const Header = () => (
+  <div>
+    <h1>Header</h1>
+  </div>
+);
+
+describe('Header', () => {
+  test('render React component', () => {
+    render(<Header />);
+    expect(screen.getByText('Header')).toBeInTheDocument();
+  });
+});
```

Comment:
- [ ] Good Job creating test files for components, but you need to test your website's components and not creating new ones, For example : 

Instead of creating a header 
```Js
 const Header = () => (
  <div>
    <h1>Header</h1>
  </div>
);
```

**_you should import your header from your components folder and test it , kindly fix this for all the newly added tests 🙏_**